worlds/mlss/Client.py

```diff
@@ -0,0 +1,266 @@
+import time
+from typing import TYPE_CHECKING, Optional, Dict, Set, List
+import struct
+from BaseClasses import MultiWorld
+from NetUtils import ClientStatus
+from .Locations import roomCount, nonBlock, beanstones, roomException, shop, badge, pants, eReward
+from .Items import items_by_id, ItemData
+from collections import defaultdict
+import sys
+import logging
+import math
+import asyncio
+
+import worlds._bizhawk as bizhawk
+from worlds._bizhawk.client import BizHawkClient
+
+if TYPE_CHECKING:
+    from worlds._bizhawk.context import BizHawkClientContext
+else:
+    BizHawkClientContext = object
```

Comment:
I should be more explicit. You can do
```py
    async def validate_rom(self, ctx: "BizHawkClientContext") -> bool:
```
where you wrap the type in quotes to prevent the issue of circular dependencies at runtime while still having functional type hints.